server/dataset/cxg_dataset.py

```diff
@@ -381,6 +381,26 @@ def get_embedding_names(self):
             raise DatasetAccessError("cxg matrix missing embeddings")
         return embeddings
 
+    def _get_filtered_cxg_schema_metadata(self) -> list[Any]:
```

Comment:
my understanding is that `obs.meta["cxg_schema"]` is just a json blob and we can filter out int64 fields here, but i'm not entirely sure what the structure of this metadata looks like